CMakeLists.txt

```diff
@@ -200,11 +200,13 @@ include(package_settings)
 # See `cmake/add_enclave.cmake` for enclave creation logic
 include(add_enclave)
 
-# User configurable options.
-option(
-  HAS_QUOTE_PROVIDER
-  "Take a build dependency on SGX DCAP, which requires FLC on target device to run."
-  ON)
+# HAS_QUOTE_PROVIDER option is deprecated.
```

Comment:
fwiw, cmake already gives a message about flags/options that aren't used, eg.

```
cmake -DFOO=BAR ..
...
  Manually-specified variables were not used by the project:

    FOO
```

Considering this was never officially supported, I think that would be acceptable too.

Reply:
Yes, I noticed this too. I'm just making it clear that it is being deprecated, since it is mentioned in contributor documents for SGX1 (non FLC).